packages/protocol/contracts/common/GoldToken.sol

```diff
@@ -152,6 +152,10 @@ contract GoldToken is
    */
   function transferFrom(address from, address to, uint256 value) external returns (bool) {
     require(to != address(0), "transfer attempted to reserved address 0x0");
+    require(
```

Comment:
Do we want to call this check every transfer ? It increases gas since we need to call external contract which needs to read from storage. 

I'm not sure if the pros are bigger than cons in this case

Reply:
Native transfers could still be used for a more gas efficient transfers.